service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -107,11 +110,14 @@ public void sendNotificationForC2Upload(final C2UploadedEvent event) {
     }
 
     @EventListener
-    public void sendNotificationForOrder(final GeneratedOrderEvent event) {
+    public void sendNotificationsForGeneratedOrder(final GeneratedOrderEvent event) {
```

Comment:
why change?

Reply:
I've added 'sendOrderIssuedNotificationToHmctsAdmin' so for clarity I thought I'd specify that this is for the GeneratedOrder event so there is a clear differentiation between a 'Generated Order' and any 'Order'. if you don't like, can revert

Reply:
please don't resolve comments before they are resolved! I would revert, it doesn't make it any clearer imo

Reply:
I didn't resolve this one, I left it open as I was waiting for response 🙂